lifetracker-api/server.js

```diff
@@ -0,0 +1,23 @@
+const express =require('express')
+const cors = require('cors')
+// const User = require('./users')
+const authRoutes = require("./routes/auth")
+const security = require('./middleware/security')
+
+const app = express()
+const PORT = 3000
+
+app.use(cors())
+app.use(express.json())
+app.use(security.extractUserFromJwt)
+
+app.use("/auth", authRoutes);
+
+app.get('/', (req,res)=> {
+    res.send({ping: 'pong'})
+})
```

Comment:
this health check should be done in `app.js` along with error handling (404 and generic catch-all)